FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/gmailapi/StandardReplyAllComposeGmailApiFlow.kt

```diff
@@ -93,8 +98,28 @@ class StandardReplyAllComposeGmailApiFlow : BaseComposeGmailFlow() {
     Thread.sleep(1000)
 
     //click on replyAll
-    onView(withId(R.id.layoutReplyAllButton))
-      .check(matches(isDisplayed()))
+    onView(
+      allOf(
+        withId(R.id.layoutReplyAllButton),
+        withParent(
+          withParent(
+            withParent(
+              hasSibling(
+                allOf(
+                  withId(R.id.layoutHeader),
+                  withChild(
+                    allOf(
+                      withId(R.id.textViewSubject),
+                      withText(SUBJECT_EXISTING_STANDARD)
+                    )
+                  )
+                )
+              )
+            )
+          )
+        )
+      )
+    ).check(matches(isDisplayed()))
```

Comment:
probably it can be moved to separate method, as similar code is used in several places, just with couple params difference - `withId(R.id.layoutReplyAllButton)` and `withText(SUBJECT_EXISTING_STANDARD)`.
something like `tapMessageActionButton(id: Int, subject: String)`, and then it'll be called as `tapMessageActionButton(R.id.layoutReplyAllButton, SUBJECT_EXISTING_STANDARD)`

Reply:
Agree. It will be for a separate PR